modules/creating-nfd-cr-cli-disconnected.adoc

```diff
@@ -0,0 +1,167 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-node-feature-discovery-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-nfd-cr-cli-disconnected_{context}"]
+= Create a NodeFeatureDiscovery CR by using the CLI in a disconnected environment
+
+As a cluster administrator, you can create a `NodeFeatureDiscovery` CR instance using the {oc-first}.
+
+.Prerequisites
+
+* You have access to an {product-title} cluster
+* You have installed the {oc-first}.
+* You have logged in as a user with `cluster-admin` privileges.
+* You have installed the the NFD Operator.
+* You have access to a mirror registry with the required images.
+* You have installed the `skopeo` CLI tool.
+
+.Procedure
+
+. Determine the digest of the registry image:
+
+.. Run the following command:
++
+[source,terminal]
+----
+$ skopeo inspect docker://registry.redhat.io/openshift4/ose-node-feature-discovery:<openshift_version>
+----
++
+.Example command
+[source,terminal]
+----
+$ skopeo inspect docker://registry.redhat.io/openshift4/ose-node-feature-discovery:v4.12
+----
+
+.. Inspect the output to identify the image digest:
++
+.Example output
+[source,terminal]
+----
+{
+  ...
+  "Digest": "sha256:1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef",
+  ...
+}
+----
+
+. Use the `skopeo` CLI tool to copy the image from `registry.redhat.io` to your mirror registry, by running the following command:
++
+[source,terminal]
+----
+skopeo copy docker://registry.redhat.io/openshift4/ose-node-feature-discovery@<image_digest> docker://<mirror_registry>/openshift4/ose-node-feature-discovery@<image_digest>
+----
++
+.Example command
+[source,terminal]
+----
+skopeo copy docker://registry.redhat.io/openshift4/ose-node-feature-discovery@sha256:1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef docker://<your-mirror-registry>/openshift4/ose-node-feature-discovery@sha256:1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef
+----
+
+. Create a `NodeFeatureDiscovery` CR:
++
+.Example `NodeFeatureDiscovery` CR
+[source,yaml,subs="attributes+"]
+----
+apiVersion: nfd.openshift.io/v1
+kind: NodeFeatureDiscovery
+metadata:
+  name: nfd-instance
+spec:
+  operand:
+    image: <mirror_registry>/openshift4/ose-node-feature-discovery@<image_digest>
```

Comment:
Does this need a callout? Or will most customers know what to enter here?

Reply:
I think it's OK since it's sort of a set value except for the user's mirror registry value and image digest value which are given as variables